containers/Dockerfile.buildenv

```diff
@@ -132,10 +133,24 @@ RUN echo 'spack load catch' >> /root/.bashrc && \
     echo 'spack load brigand' >> /root/.bashrc && \
     echo 'spack load blaze' >> /root/.bashrc && \
     echo 'spack load gsl' >> /root/.bashrc && \
+    echo 'spack load fftw' >> /root/.bashrc && \
     echo 'spack load libxsmm' >> /root/.bashrc && \
     echo 'spack load yaml-cpp' >> /root/.bashrc && \
     echo 'spack load benchmark' >> /root/.bashrc
 
+# Download and build SSHT, install to /usr
```

Comment:
I would ideally have SSHT be part of spack. If you don't want to put in a PR with them, I can do it. They typically are quite fast about getting new packages in :)

Reply:
sure, I can make that happen.